<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>449</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>book.svg</normaloff>book.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Rockwell Extra Bold</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LIBRARY MANAGEMENT SYSTEM</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Catalogue</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>If you want to choose a book, double-click on it!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add a book</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete a book</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rent a book</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Make reservation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Renew a book</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_9">
      <property name="font">
       <font>
        <family>Rockwell</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log out</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>released()</signal>
   <receiver>tableView</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
